ceph/rados/core_workflows.py

```diff
@@ -751,6 +761,38 @@ def enable_balancer(self, **kwargs) -> bool:
         log.info(f"the balancer status is \n {out}")
         return True
 
+    def get_read_scores_on_cluster(self) -> dict:
```

Comment:
I think we can enhance this module or write a new one to fetch the read score for a particular pool. Reason being in the test case, there are at least 2 instances where we are fetching the read score for pool `test-pool`, but for doing so we are first gathering the stats for all the pools from the cluster

Reply:
Sounds Good. I'll add a backlog task to make changes for the method.